spisea/evolution.py

```diff
@@ -1615,6 +1615,91 @@ def get_orig_pisa_isochrones(metallicity=0.015):
 
     return data
 
+
+#==============================#
+# BPASS v2.2 models
+#==============================#
+#These models were published by Stanway and Elridge et. al in 2018
+#The code is based from the Merged Evolution model methods outlined in the . After all, I want some degree of consistency with regards to
+#Our original isochrone files are the stellar models (single, binary, and secondary) listed in the BPASS input files from one of the stellar models for ages 10**6.0, 10**6.1, ... , 10**11.0 years.
+
+class BPASS (StellarEvolution):
+    def __init__(self):
+        #Note: All of the possible mass list items include possible mass for single star, binary primary and secondary.
+        self.age_list=[round(6.0+x*0.1,1) for x in range(0, 51)]
+        #Possible metallicities for BPASS models
+        self.z_list=[10**-5, 10**-4,10**-3, 0.002, 0.003, 0.004, 0.006, 0.008, 0.010, 0.014, 0.020, 0.030, 0.040]
+        self.models_dir='/g/lu/models/evolution/BPASS/v2.2/'
+        #Possible masses
+        self.mass_list=[round(0.1, 1)]+[round(0.12+x*0.02, 2) for x in range(95)]+[round(2.05+x*0.05, 2) for x in range(20)]+[round(3.1+0.1*x,1) for x in range(70)]
+        self.mass_list=self.mass_list+[11+x for x in range(90)]+[125+25*x for x in range(8)]+[400]+[500]
+        self.z_solar=0.02
+        self.z_file_map={10**-5: 'zem5/', 10**-4: 'zem4/', 0.003:'z003/',0.002: 'z002/', 0.003: 'z003/', 0.004: 'z004/', 0.006:
+                         'z006/',0.008: 'z008/', 0.010:'z010/',0.014:'z014/', 0.020: 'z020/', 0.030: 'z030/',0.040:'z040/' }
+    def isochrone(self, filePath="", age=1*10**8, metallicity=0.0, binary=True):
```

Comment:
We shouldn't need a filePath... lets hardcode to a specific sub-directory structure under the environment variable specified.